src/guides/appchain-evm.md

```diff
@@ -1,9 +1,162 @@
 ## EVM Compatible Appchain
 
-The [Barnacle EVM](https://github.com/octopus-network/barnacle/tree/release-v0.9.13-evm) is an EVM compatible appchain development template provided by Octopus Network. It uses the [Frontier](https://github.com/paritytech/frontier) developed by Parity, and its functionality is basically the same as [Moonbeam Network](https://moonbeam.network/).
+The [Barnacle EVM](https://github.com/octopus-network/barnacle/tree/release-v0.9.13-evm) is an EVM-compatible Appchain development template provided by Octopus Network. It uses the [Frontier](https://github.com/paritytech/frontier) developed by Parity, and its functionality is the same as [Moonbeam Network](https://moonbeam.network/).
```

Comment:
Actually, we choose a solution different from Moonbeam, which is considering the good experience of EVM-compatible Appchain's validators and app users. App users use EVM toolsets like MetaMask, but validators still can use the polkadot.js plugin wallet.

So please remove the below words:
1. ", and its functionality is the same as [Moonbeam Network](https://moonbeam.network/)";
2. and functions similar to the Moonbeam network;
3. Barnacle EVM is similar to Moonbeam because it is a Substrate-based EVM.

Reply:
I have removed the unnecessary words.